lib/features/create/Create.js

```diff
@@ -239,9 +242,11 @@ export default function Create(
       });
     } else {
       // invoke create, if connect is not set
-      shape = modeling.createShape(shape, position, target, {
-        attach: attach
-      });
+      shape = modeling.createShape(shape, position, target, assign(
+        hints, {
```

Comment:
Please create a new, merged object safely by passing an empty object as the first argument: `assign({}, hints, { ... })`.

Reply:
Could we make hint passing consistent and add it in the `modeling#appendShape` case, too?

Reply:
Yes we could, but context hint passing is different in [Modeling#appendShape](https://github.com/bpmn-io/diagram-js/blob/487ffd40b984d94e4d886db109a216cfe0442caa/lib/features/modeling/Modeling.js#L337-L339) compared to [Modeling#createShape](https://github.com/bpmn-io/diagram-js/blob/487ffd40b984d94e4d886db109a216cfe0442caa/lib/features/modeling/Modeling.js#L285). 

If we want to include all hints in `appendShape`'s context like we do in `createShape`, it will result in a breaking change. 

Is that something we want to handle in this PR?

Reply:
How is it going to be a breaking change?

Reply:
If we want hints to be consistent in `modeling#appendShape` as they are now in `modeling#createShape`, the `connection`, `connectionParent` and `attach` properties would no longer be available directly on `context` but on `context.hints`.

So any code that calls something like `context.connection` will break after that change.